### DISCUSSION TITLE
[Vue warn]: Component <Anonymous> is missing template or render function.

### DISCUSSION THREAD

**Discussion by baronkoko** [nuxt/nuxt#19020] [Questions]
### Environment
------------------------------
- Operating System: `Darwin`
- Node Version:     `v16.14.0`
- Nuxt Version:     `3.0.0`
- Nitro Version:    `1.0.0`
- Package Manager:  `npm@9.3.1`
- Builder:          `vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`
------------------------------

### Reproduction
https://stackblitz.com/edit/github-tpgvk1?file=app.vue

### Describe the bug
If I have an async component in `app.vue` and after a request via the `useAsyncData` I throw `createError`, then the Vue warnings appear on the error page. Can someone help with this, please?
<img width="643" alt="image" src="https://user-images.githubusercontent.com/27996386/218717293-cec255d8-3a7a-4bbc-92e1-954547dfc80d.png">

**Comment by Baroshem** [comment]
Hey,

Basically, when the resource cannot be found ( a return code is 404) the bug appears). Once I switched to something that actually exist, it works.

i.e.

```vue
<template>
  <VModal />
</template>

<script lang="ts" setup>
import { defineAsyncComponent } from 'vue';
import { createError } from '#imports';

const VModal = defineAsyncComponent(() => import('~/components/VModal.vue'));

const { error } = await useAsyncData('mountains', () =>
  $fetch('https://rickandmortyapi.com/api')
);

if (error.value) {
  throw createError({
    message: error.value?.message,
  });
}
```

Also, I think you should be using `statusMessage` instead of `message` as instructed in the docs [here](https://nuxt.com/docs/api/utils/create-error)

**Reply by baronkoko** [reply]
@Baroshem ty for the reply. But that's the problem, in my example, the wrong URL is set intentionally, in order to have the `error.value`  and call the `createError`, because these warnings occur precisely on the `error.vue` page

**Reply by baronkoko** [reply]
Still happens in Nuxt `3.2.2`
